projects/hslayers/src/components/add-data/common/common-file.service.ts

```diff
@@ -76,6 +80,36 @@ export class HsAddDataCommonFileService {
     this.hsLaymanService.totalProgress = 0;
   }
 
+  /**
+   * Check if provided url exists and is obtainable
+   * @param url - Provided url
+   * @param app - App identifier
+   */
+  async isUrlObtainable(url: string, app: string): Promise<boolean> {
+    if (!url || url === '') {
+      return;
+    }
+    url = this.hsUtilsService.proxify(url, app);
+    try {
+      const response = await fetch(url, {
```

Comment:
Is there any (dis)advantage of using `fetch` instead of Angular's `HttpClient`?

Reply:
Well using fetch code is more simple and i don't have to subscribe to get response. Also when using URL like for KML, that requests file download (XMLHttpRequest), I struggled to get httpClient to work. Fetch just made it more simple :D

Reply:
If you are curious about the differences: https://stackoverflow.com/questions/53660262/angular-http-vs-fetch-api . I am fine with `fetch` in this use-case.